### PR TITLE
Mobile webpage1

### PR DESCRIPTION
**Pull request by Lockless404**
Finished with the first mobile page.

### CODE REVIEW

**Comment by Lockless404** [comment]
Completed first Mobile page. The topbar, Heading, description and social media.

File: portfolio.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  <header>
+    <a href="" class="nameLink">John Doe</a>
+    <nav>
+      <img src="images/Icon-Menu.png" alt="Menu Icon" class='menuIcon'>
+    </nav>
+  </header>
+  <h1>Hey There. I'm Alex</h1>
+  <h2>I’am a Software Developer</h2>
+  <h3>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.</h3>
```
Comment (DanSam5K) [inline portfolio.html:20]: - [ ] How about using a (p tag element) and changing the fonts with style instead? 🤔

**Review by DanSam5K — CHANGES_REQUESTED** [review]
# Hi @Lockless404  👏 🥇 , 

## Good job so far! 🌱 
You really did an amazing job and almost 🔨 nailed it first try 🤗 
## Highlights 💥 💥 
- Your commit message is great and amazing you handle it like a pro✌️ 
- All your linters checks passed 👍 🥇 
- You stick to the general requirements pretty good 👍 

But 😉  there are some issues that you still need to work on to go to the next project but you are almost there! ❤️ 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._
- [ ] Please remember to wrap your headlines inside a (section tag)
- [ ] Also please remember the list of icons should use a ul tag to group them, and li per icon; each icon has a link (a tag)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] [Optional] Improving your documentation with a procedure on how one can get a clone of your repo into their local machine. 😉 
 > I can't wait to see the final implementation of this project because it really promises to be a great one 💯 
## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <h1>Hey There. I'm Alex</h1>
+    <h2>I’am a Software Developer</h2>
+    <p class="firstParagraph">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</p>
+    <ul class="Icon">
+      <li><img src="images/Icon-GitHub.png" alt="Github Icon" class="Icons Icon1"></li>
+      <li><img src="images/Icon-Linkedin.png" alt="Linkedin Icon" class="Icons Icon2"></li>
+      <li><img src="images/Icon-Angelist.png" alt="Angelist Icon" class="Icons Icon3"></li>
+      <li><img src="images/Icon-Twitter.png" alt="Twitter Icon" class="Icons Icon4"></li>
+      <li><img src="images/Icon-Medium.png" alt="Medium Icon" class="Icons Icon5"></li>
```
Comment (SajjadAhmad14) [inline portfolio.html:27]: Please wrap every icon image with a link tag so that it can be a clickable link.

File: portfolio.css [hunk 3]
```diff
@@ -0,0 +1,137 @@
+body {
```
Comment (SajjadAhmad14) [inline portfolio.css:1]: [ OPTIONAL ] The page has a horizontal scroll. It means that some content is overflowing out of its container. Please fix the scroll as it is not considered good for the design.
scroll

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Hamza Tarar, 
 ### Thank you for implementing changes suggested previously :ok_hand:

## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 4]
```diff
@@ -1,29 +1,22 @@
...
-- Technologies used
+- HTML,CSS
+- No Frameworks
+- No Technologies
 
 ## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
+images/demo.png
```
Comment (jmontanholi) [inline README.md:16]: # [OPTIONAL]
- [ ] You don't have a live demo link, consider either deploying the project with GitHub pages or removing this section

File: README.md [hunk 5]
```diff
@@ -1,29 +1,22 @@
...
 
 ## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
 
 To get a local copy up and running follow these simple example steps.
```
Comment (jmontanholi) [inline README.md:22]: # [OPTIONAL]
- [ ] This section should include instructions to other people on how they can set up this project locally, this way they can open your project on their computer and look at the code in their code editor. 👍🏻

**Review by jmontanholi — APPROVED** [review]
Hello Hamza Tarar!

Congratulations!! You have done a great job on this Project. Your PR is ready to merge so you can continue being great!

Cheers and Happy coding!👏👏👏
# [OPTIONAL]
Although this is optional I highly recommend you to do it in order to have a more professional project.

- [ ] Please consider improving your Readme file, there are some empty sections there and it lacks some information. (Look at the comments)
- [ ] Consider doing your project for mobile sizes already, your background image is outside its place and if you use the browser console to look at the project you'll see that it is a bit weird. Try using the console to develop directly for mobile sizes. 👍🏻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**